tests/kafkatest/services/security/security_config.py

```diff
@@ -110,7 +111,7 @@ def __init__(self, security_protocol, interbroker_security_protocol=None, sasl_m
     def client_config(self, template_props=""):
         return SecurityConfig(self.security_protocol, sasl_mechanism=self.sasl_mechanism, template_props=template_props)
 
-    def setup_node(self, node):
+    def setup_node(self, node, miniKdc=None):
```

Comment:
Does it make sense to set `self.kdc` in the constructor? And then having non-None self.kdc would be part of the logic in `has_sasl_kerberos`